home/pi/setup.sh

```diff
@@ -49,39 +57,53 @@ touch ~/.hushlogin
 # GUI: Install plymouth
 git clone https://github.com/forslund/mycroft-plymouth-theme
 cd mycroft-plymouth-theme
-./install.sh
-cd ..
+echo -n "Press any key? Okay!" | ./install.sh
+cd ~
 sudo plymouth-set-default-theme mycroft-plymouth-theme
 
 # Volume: Install I2C support (might require raspi-config changes first)
-sudo apt-get install i2c-tools
-
-# Create basic folder structures
-mkdir ~/bin
+sudo apt-get install -y i2c-tools
+sudo raspi-config nonint do_i2c 0
 
 # Get the Picroft conf file
 cd /etc/mycroft
 sudo wget -N $REPO_PATH/etc/mycroft/mycroft.conf
+cd ~
+
+wget -N $REPO_PATH/home/pi/.bashrc
+wget -N $REPO_PATH/home/pi/auto_run.sh
+wget -N $REPO_PATH/home/pi/mycroft.fb
 
-echo "Downloading 'mycroft-core'..."
+mkdir -p ~/bin
+cd ~/bin
+wget -N $REPO_PATH/home/pi/bin/mycroft-wipe
+chmod +x mycroft-wipe
 cd ~
+
+# mycroft-core
 git clone https://github.com/MycroftAI/mycroft-core.git
 cd mycroft-core
-git checkout dev 
-
-echo
-echo "Beginning building mycroft-core.  This'll take a bit.  Answer: Y Y N to questions"
-echo "then take a break for an hour!  Results will be in the ~/build.log"
 IS_TRAVIS=true bash dev_setup.sh 2>&1 | tee ../build.log
-echo "Build complete.  Press any key to review the output before it is deleted."
-read -N1 -s key
-nano ../build.log
-rm ../build.log
+# Keep for now.
+#rm ../build.log
+cd ~
 
-echo
-echo "Retrieving default skills"
-sudo chown -R pi:pi /opt/mycroft
+# Streaming STT
+pip install google-cloud-speech
+# Insert stt key, remove placeholder comment, format and write to file.
+sed '/# Google Service Key/r /boot/stt.json' /etc/mycroft/mycroft.conf \
+    | sed 's|# Google Service.*||' \
+    | python -m json.tool \
+    | sudo tee /etc/mycroft/mycroft.conf
+
+# skills
 ~/mycroft-core/bin/mycroft-msm default
```

Comment:
When msm 0.7.9 hits this should be changed to ~/mycroft-core/bin/mycroft-msm -p mycroft_mark_2pi default to pick the defaults from the correct list.

Reply:
Sounds good. I'll test that when it is ready